.travis.yml

```diff
@@ -1,37 +1,44 @@
 sudo: required
-
 language: node_js
 node_js:
-  - "4.4.4"
+- 4.4.4
```

Comment:
Blah.  That happened when I used `travis encrypt --yes`.  I'll set it back to how it was